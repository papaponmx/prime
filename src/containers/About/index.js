import React from 'react'

const About = () => (
  <div>
    <h1>About Page</h1>
    <p>TODO: Create real content for this page.</p>
  </div>
)

export default About
