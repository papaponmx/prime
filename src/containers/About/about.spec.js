import React from 'react';
import About from '.';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });

describe('About', () => {
  it('Sould Render about page', () => { Â 
    const output = shallow(
      <About />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });


});
