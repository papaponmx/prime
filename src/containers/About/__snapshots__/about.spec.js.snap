// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Sould Render about page 1`] = `
<div>
  <h1>
    About Page
  </h1>
  <p>
    TODO: Create real content for this page.
  </p>
</div>
`;
